README.md

```diff
@@ -12,29 +12,39 @@ addSbtPlugin("nl.codestar" % "sbt-findsecbugs" % "(current version)")`
 
 (You can find the current version [here](https://github.com/code-star/sbt-findsecbugs/releases).)
 
+Optionally, override the default SpotBugs and FindSecBugs versions in your `build.sbt`:
+
+```
+findSecBugsSpotBugsVersion := "4.9.4"
+findSecBugsSpotBugsPluginVersion := "1.14.0"
+```
+
```

Comment:
Maybe also put version badges here for easier reference, like in the original PR: https://github.com/code-star/sbt-findsecbugs/pull/41/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R21 ?